task_12/src/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Pizza App</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+	<header class="header">
+		<div class="header__container">
+			<div class="header__time">
+				<i class="fa fa-clock-o"></i>
+				<time class="time__current-time" id="current-time"></time>
+			</div>
+			<div class="header__logo">
+				<a href="#">
+					<img class="logo__img" src="img/pizza-logo.png" alt="Pizza app logo">
+				</a>
+			</div>
+			<div class="header__login">
+				<button type="button" class="button login__button">
+					<i class="fa fa-sign-in"></i>
+					Login/Signup
+				</button>
+			</div>
+		</div>
+	</header>
+
+	<main class="main">
+		<button class="button main__button">+ Add new pizza</button>
+		<div class="main__container">
+
+			<article class="main__item">
+				<img src="img/item__pizza_2x.png" alt="" class="item__pic">
```

Comment:
You should provide heading element in each _section_, which actually can be a `<section>` or an `<article>`.